src/pages/Transfer/CrossChainTransferForm/index.tsx

```diff
@@ -0,0 +1,59 @@
+
+import { useTranslation } from 'react-i18next';
+import { withErrorBoundary } from 'react-error-boundary';
+import { useSelector } from 'react-redux';
+
+import { COLLATERAL_TOKEN_SYMBOL } from 'config/relay-chains';
+import { StoreType } from 'common/types/util.types';
+import { displayMonetaryAmount } from 'common/utils/utils';
+import AvailableBalanceUI from 'components/AvailableBalanceUI';
+import Accounts from 'components/Accounts';
+import Chains from 'components/Chains';
+import TokenField from 'components/TokenField';
+import ErrorFallback from 'components/ErrorFallback';
+import FormTitle from 'components/FormTitle';
+import SubmitButton from 'components/SubmitButton';
```

Comment:
```suggestion
import AvailableBalanceUI from 'components/AvailableBalanceUI';
import Accounts from 'components/Accounts';
import Chains from 'components/Chains';
import TokenField from 'components/TokenField';
import ErrorFallback from 'components/ErrorFallback';
import FormTitle from 'components/FormTitle';
import SubmitButton from 'components/SubmitButton';
import { COLLATERAL_TOKEN_SYMBOL } from 'config/relay-chains';
import { displayMonetaryAmount } from 'common/utils/utils';
import { StoreType } from 'common/types/util.types';
```

Reply:
Yes, no problem. I've added a note to the linting ticket to use `sortImports` as we shouldn't be managing things like this manually.

Reply:
@tomjeatt 
Oh, can we do this automatically using ESLinting? That's good.